app/components/edit/supplemental_files_step_component.html.erb

```diff
@@ -9,13 +9,15 @@
 
   <% component.with_body_content do %>
     <%= render Shared::SupplementalFilesStepBodyHelpComponent.new %>
-      <% if supplemental_files.attached? %>
+      <% unless supplemental_files.empty? %>
```

Comment:
I might have naïvely expected these to do the same thing. Can you say more about this change?

Reply:
Aha, thank you! I'm primarily wondering if we could stick with the `if` here. Maybe:

```ruby
      <% if supplemental_files.present? %>
```

or

```ruby
      <% if supplemental_files.any? %>
```

❓